### PR TITLE
Leo Editor recipe (1st conda-forge attempt)

### PR DESCRIPTION
**Pull request by maphew**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

----

The url is currently set to a specific commit and not a release. After I know everything in CI is working and there are no more conda forge changes to make to the source repository we'll make a release.

### CODE REVIEW

File: recipes/leo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+app:
+    entry: Leo Editor
+    summary: "Leo is an Outliner, Editor, and Personal Information Manager written in 100% Python."
+    icon: LeoApp.ico
+
+about:
+  home: "https://leoeditor.com"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (isuruf) [inline recipes/leo/meta.yaml:52]:
```suggestion
  license_file:
    - LICENSE
    - leo/external/ckeditor/LICENSE.md
```

File: recipes/leo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - leo
+
+app:
+    entry: Leo Editor
+    summary: "Leo is an Outliner, Editor, and Personal Information Manager written in 100% Python."
+    icon: LeoApp.ico
+
+about:
+  home: "https://leoeditor.com"
+  license: MIT
```
Comment (isuruf) [inline recipes/leo/meta.yaml:50]:
```suggestion
  license: MIT AND (MPL-1.1 OR MPL-2.0 OR LGPL-2.1-or-later OR GPL-2.0-or-later)
```

File: recipes/leo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - leo
+
+app:
+    entry: Leo Editor
+    summary: "Leo is an Outliner, Editor, and Personal Information Manager written in 100% Python."
+    icon: LeoApp.ico
+
+about:
+  home: "https://leoeditor.com"
+  license: MIT AND (MPL-1.1 OR MPL-2.0 OR LGPL-2.1-or-later OR GPL-2.0-or-later)
```
Comment (isuruf) [inline recipes/leo/meta.yaml:50]:
```suggestion
  license: MIT AND (MPL-1.1 OR MPL-2.0 OR LGPL-2.1-or-later OR GPL-2.0-or-later) AND (BSD-2-Clause)
```

File: recipes/leo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  license_family: MIT
```
Comment (isuruf) [inline recipes/leo/meta.yaml:51]:
```suggestion
  license_family: Other
```

**Comment by isuruf** [comment]
Have you looked at all the external files and made sure their licenses are attributed? I haven't looked at it yet.

**Comment by maphew** [comment]
Yes the only remaining 3rd party modules are _ckeditor_ and _npyscreen_, and both have license files now.

**Comment by maphew** [comment]
Oh wait, one more commit incoming in next few minutes, to update the download url and hash.

**Pull request closed without merging**